### ISSUE TITLE
Is this project alive ?

### ISSUE THREAD

**Issue by silviucpp** [erlang-lager/lager#570]
Hello,

I see very little activity in this project in the last time. Is the project still alive ?

Silviu

**Comment by jadeallenx** [comment]
Yes, it's still alive. It's just a pretty mature project at this point and not under a lot of active development at the moment.

**Comment by silviucpp** [comment]
@jadeallenx There are important pull requests that are not moving for more than 2 years.. Like https://github.com/erlang-lager/lager/pull/524

**Comment by jadeallenx** [comment]
Like I said, it's a mature project that is not under a lot of development _at this time_. The PR you reference is more of a "draft" PR than something that's finished and awaiting review/merging. I discussed it with Andrew extensively at the time. When he has bandwidth or someone else has the itch to scratch, they can pick it up.

**Comment by elsbiet** [comment]
Erlang/OTP 29 - there are lots of warnings:

_build/default/lib/lager/src/lager_app.erl:285:6: Warning: 'catch ...' is deprecated; please use 'try ... catch ... end' instead.
Compile directive 'nowarn_deprecated_catch' can be used to suppress
warnings in selected modules.
%  285|     ?INT_LOG(notice, "Deprecated lager_file_backend config detected, please consider updating it", []),
%     |      ^

and

 _build/default/lib/goldrush/src/glc_run.erl:
    \x{2502}
 15 \x{2502}      case (catch Fun(Event, Store)) of
    \x{2502}            \x{2570}\x{2500}\x{2500} Warning: 'catch ...' is deprecated; please use 'try ... catch ... end' instead.
Compile directive 'nowarn_deprecated_catch' can be used to suppress
warnings in selected modules.

question: are there any efforts to make lager compatible to Erlang/OTP 29?

**Comment by Vagabond** [comment]
I'm still using it daily with OTP 28, on the https://github.com/erlang-lager/lager/pull/524 branch, but I have not tried it with OTP 29 yet.

**Comment by jadeallenx** [comment]
I haven't touched Erlang in almost four years now, but I'm happy to add maintainers to the project if people want to step up.
